### DISCUSSION TITLE
Can you add the ability to change the icon?

### DISCUSSION THREAD

**Discussion by taisontan** [homebridge-plugins/homebridge-august#97] [Ideas]
Currently, the only option is the LOCK icon 🔒.
It'd be nice to have the option of choosing a door icon🚪.

**Comment by donavanbecker** [comment]
Which icon?

**Reply by taisontan** [reply]
This is the icon. Love the plugin. It made the August lock actually functional with HomeKit. 

![IMG_2819](https://github.com/donavanbecker/homebridge-august/assets/158862952/db657a66-9a8a-45da-9e00-87ad2c855d72)

**Reply by donavanbecker** [reply]
Nothing I can do with that. This is functionality with the Home App. You can change it to another lock icon though.

![image](https://github.com/donavanbecker/homebridge-august/assets/9875439/a559bb51-7fd9-467f-ae9a-885998121e62)

**Reply by taisontan** [reply]
Oh how did you do that? I tried to tap on the icon but it won’t let me change it

**Reply by donavanbecker** [reply]
go into accessory settings, if you have contact sensor enabled, you will need to click into the accessories
<img width="510" alt="Screenshot 2024-02-08 at 1 47 37 AM" src="https://github.com/donavanbecker/homebridge-august/assets/9875439/e6eeb949-e8b7-4e15-89c9-d3a7086eabf8">
then click into the lock and you will be able to change the image.

**Reply by taisontan** [reply]
Thank you!!!!
I didn’t think of having to go another level deeper due to having the contact sensor enabled. This is exactly what I wanted, but didn’t realize it. This is what the icon was when I was using the native August HomeKit. I just didn’t remember it. Thank you!
